devtools-html/debugger.html
# Show open tabs

Associated Issue: #<issue number>

### Summary of Changes

* change 1
* change 2

### Test Plan

- Open some files
- cmd+p to open quick open
- see that opened tabs are the listed entries
- close all tabs
- cmd+p to open quick open
- see that first 100 sources are the listed entries
